lib/components/mobile/navigation-bar.js

```diff
@@ -48,11 +49,9 @@ class MobileNavigationBar extends Component {
           <Navbar.Brand>
             {showBackButton ? (
               <div className="mobile-back">
-                <Icon
-                  fixedWidth={false}
-                  name="arrow-left"
-                  onClick={this._backButtonPressed}
-                />
+                <StyledIconWrapper onClick={this._backButtonPressed}>
```

Comment:
I don't think it's necessary on this because it's a single icon that is not getting aligned vertically with other icons. The mobile navbar really needs some work, though, there's a lot of magic numbers of padding and fixed positions in there and it doesn't look great. I think that needs to be a separate PR, though.

Reply:
Could we do that separate PR as a piggyback PR on this one? I don't like the idea of a PR merge breaking things

Reply:
I'm happy to report this is not a regression, as it looks just as broken on dev.